gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/JWTFederationFilter.java

```diff
@@ -90,6 +94,14 @@ public void init( FilterConfig filterConfig ) throws ServletException {
       expectedJWKSUrl = oidcjwksurl;
     }
 
+    allowedJwsTypes = new HashSet<>();
+    final String allowedTypes = filterConfig.getInitParameter(ALLOWED_JWS_TYPES);
+    if (allowedTypes != null) {
+      Stream.of(allowedTypes.trim().split(",", -1)).forEach(allowedType -> allowedJwsTypes.add(new JOSEObjectType(allowedType.trim())));
```

Comment:
nit, -1 limit for split appears redundant.

Reply:
Nice catch! Fixed :)